harvard-university-for-the-creative-arts.csl

```diff
@@ -110,11 +125,11 @@
           </else-if>
         </choose>
       </if>
-      <else-if type="manuscript song post-weblog post personal_communication speech" match="any">
-        <group delimiter=". ">
+      <else-if type="manuscript song post-weblog post" match="any">
+        <group delimiter=" ">
           <text variable="genre" prefix="[" suffix="]"/>
           <text variable="medium" prefix="[" suffix="]"/>
-          <text variable="note" prefix="[" suffix="]"/>
+          <text variable="note" suffix="."/>
```

Comment:
> Is there another way to do this?

You're using Zotero, right? In that case, it does look like you can't use "song" for Podcast since it lacks any date variables other than "accessed" (https://aurimasv.github.io/z2csl/typeMap.xml#map-podcast). There is an old Zotero ticket (https://github.com/citation-style-language/zotero-bits/issues/2) to add the CSL "issued" date variable to Zotero's podcast item type. Not sure if there are any other workarounds right now.

> Alternatively I can take it out and if you'd rather it wasn't there.

I think so, yes. We try to avoid having any reference-manager-specific hacks (like this one for Zotero) in our CSL repository styles, and the use of "note" is problematic because users may already use it for other purposes.